File: 3790-fruits-into-baskets-ii/3790-fruits-into-baskets-ii.java
class Solution {
    public int numOfUnplacedFruits(int[] fruits, int[] baskets) {
        int count = fruits.length ; 
        boolean[] took = new boolean[baskets.length];
        for(int i = 0 ; i < fruits.length ; i++){
            for(int j = 0 ; j < baskets.length ; j++){
                if(!took[j] && baskets[j] >= fruits[i]){
                    took[j] = true ; 
                    count-- ; 
                    break ; 
                }
            }
        }
        return count ; 
    }
}

File: 3790-fruits-into-baskets-ii/3790-fruits-into-baskets-ii.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        used = set()
        unplaced = 0
        for fruit in fruits:
            placed = False
            for i in range(len(baskets)):
                if i not in used and baskets[i] >= fruit:
                    used.add(i)
                    placed = True
                    break
            if not placed:
                unplaced += 1
        return unplaced
